rabbitmq/ci/start-docker.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source utils/bash/ci.sh
+
+NAME='dd-test-rabbitmq'
+
+stop-docker $NAME
```

Comment:
Isn't it safe to assume the container will not exist, and if it exists, like when we do a `SKIP_CLEANUP=1` just remove manually. In travis CI this should normally be a NOP.